xml/art-obs-beginners-guide.xml

```diff
@@ -175,24 +167,28 @@ https://www.suse.com/communities/blog/suse-studio-integration/
    <title>Projects and Packages</title>
    <para>
     In &obsa;, packages are organized in <emphasis>projects</emphasis>.
-    A single project can contain several packages and serves a specific
-    purpose, defines access control, other repositories, and build targets
-    (operating systems and architectures).
+    A single project can contain several packages and usually serves a specific
+    organizational purpose. Generic access control, related repositories, and
+    build targets (operating systems and architectures) are all defined at
+    the project level.
    </para>
    <para>
     Projects can also contain other projects (subprojects) to structure work.
     Each project name is separated by colons.
```

Comment:
I would at least mention that maintainers from upper level can gain write access to subprojects. This  is in the end the entire point of the sub project structure and also a security matter if you are not aware of that.